### PR TITLE
Fixes #162 Fixing up tests to use assertCountEquals for list values

### LINKED ISSUE #162: test_student_class_site_view is failing
One test in `seumich/tests.py` seems to be failing: `test_student_class_site_view`. My cursory look at the code did not provide any clues as to what might be going on.

Screen Shot 2019-10-03 at 11 03 41 AM

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
Looks like the main problem here was that the ordering changed, probably with the python upgrade and this was never checked. 

This didn't seem like a great way to be checking these values, so I changed it to assertCountEqual. I tried a few things that *broke* this test (changing/removing values) and it failed as expected.

**Comment by jonespm** [comment]
Also this test case wasn't using the values from settings, so it wasn't working for me when I tried to test this in docker. 

I was testing this quickly with 
`docker exec -it student_explorer /bin/bash -c "echo yes | python manage.py test seumich.tests.SeumichTest.test_student_class_site_view"`

File: seumich/tests.py [hunk 1]
```diff
@@ -572,17 +571,17 @@ def test_student_class_site_view(self):
...
-                                 [("{'color': '#255c91', 'values': [[1, 0L], "
-                                   "[2, 0L], [3, 0L], [4, 0L], [5, 65L], "
-                                   "[6, 68L], [7, 68L], [8, 68L], [9], [10], "
-                                   "[11], [12], [13], [14]], "
-                                   "'key': 'Student'}"),
-                                  ("{'color': '#F0D654', 'values': [[1, 0L], "
-                                   "[2, 0L], [3, 0L], [4, 0L], [5, 58L], "
-                                   "[6, 58L], [7, 58L], [8, 57L], [9], [10], "
-                                   "[11], [12], [13], [14]], 'key': 'Class'}")
-                                  ])
```
Comment (ssciolla) [inline seumich/tests.py]: I'm not crazy about including the commented out line in committed code, but I'll defer to you. If you remove it, let's also remove `import sys`, as that's throwing a Codacy error.

Comment (jonespm) [reply seumich/tests.py]: Ok yeah I'll remove it. Hopefully we'll remember if we ever need to try to debug this in the future. Maybe we can add a better way to debug running these unit tests, like using ptvsd or something too.

File: student_explorer/settings.py [hunk 2]
```diff
@@ -380,4 +380,4 @@ def show_debug_toolbar(request):
 }
 
 # These are the settings exported from this file to templates
```
[the remark this answers is not in the document]
Comment (jonespm) [reply student_explorer/settings.py]: I don't think this was, but it was actually a warning that came up in the unit test from old code in index.html. I don't think it's actually used as I've never seen a login URL anywhere, it just goes straight to the login page. Maybe it's not needed.

I added this module https://pypi.org/project/django-settings-export/ that allows explicit exporting of settings and this template was using settings.LOGIN_URL. 

I think it's more of just old code that needs to get cleaned up but was causing an error for the tests.

**Pull request merged**